backend/src/main/resources/heroku.properties

```diff
@@ -0,0 +1,4 @@
+jwt.secret=${JWT_SECRET}
+spring.datasource.url=${DATABASE_URL}
```

Comment:
Is this file used anywhere? Or is it picked up automatically by Heroku?

Reply:
Heroku uses the Procfile to override application.properties with this.